etc/kayobe/ansible/build-ofed.yml

```
@@ -0,0 +1,103 @@
+---
+- name: Build OFED packages
+  become: true
+  hosts: ofed-builder
+  gather_facts: false
+  tasks:
+    - name: Extend the home logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_home
+        size: +2G
+        resizefs: true
+
+    - name: Extend the temporary logical volume
+      community.general.lvol:
+        vg: rootvg
+        lv: lv_var_tmp
+        size: +2G
+        resizefs: true
+
+    - name: Install package dependencies
+      ansible.builtin.dnf:
+        name:
+          - kpartx
+          - perl
+          - rpm-build
+          - automake
+          - patch
+          - kernel
+          - kernel-devel
+          - autoconf
+          - pciutils
+          - kernel-modules-extra
+          - kernel-rpm-macros
+          - lsof
+          - libtool
+          - tk
+          - gcc-gfortran
+          - tcl
+          - createrepo
+          - cmake-filesystem
+          - libnl3-devel
+          - python3-devel
+        state: latest
+        update_cache: true
+
+    - name: Set GRUB default index to saved
+      ansible.builtin.lineinfile:
+        path: /etc/default/grub
+        line: "GRUB_DEFAULT=saved"
+
+    - name: Set update default index in kernel sysconfig
+      ansible.builtin.lineinfile:
+        path: /etc/sysconfig/kernel
+        line: "UPDATEDEFAULT=yes"
+        create: true
+
+    - name: Update the default kernel entry
+      ansible.builtin.shell:
+        cmd: |
+          grubby --set-default /boot/$(rpm -qa kernel-devel | sed 's/kernel-devel/vmlinuz/g')
+          grub2-mkconfig -o /boot/grub2/grub.cfg
+
+    - name: Disable noexec in temporary file system
+      ansible.builtin.replace:
+        path: /etc/fstab
+        regexp: 'noexec,'
+
+    - name: Reboot builder to apply kernel update
+      ansible.builtin.reboot:
+        reboot_timeout: 600
+
+    - name: Add DOCA host repository
+      ansible.builtin.dnf:
+        name: https://developer.nvidia.com/downloads/networking/secure/doca-sdk/DOCA_2.8/doca-host-2.8.0-204000_{{ stackhpc_pulp_doca_ofed_version }}_rhel9{{ stackhpc_pulp_repo_rocky_9_minor_version }}.x86_64.rpm
+        disable_gpg_check: true
+
+    - name: Install DOCA extra packages
+      ansible.builtin.dnf:
+        name: doca-extra
+
+    - name: Create build directory
+      ansible.builtin.file:
+        path: /home/cloud-user/ofed
+        state: directory
+        mode: 0777
+
+    - name: Set build directory
+      ansible.builtin.lineinfile:
+        path: /opt/mellanox/doca/tools/doca-kernel-support
+        search_string: 'TMP_DIR=$1'
+        line: '    TMP_DIR=/home/cloud-user/ofed'
```

Is that whitespace required?
```suggestion
        line: 'TMP_DIR=/home/cloud-user/ofed'
```

---

For indentation yes, though maybe we can use replace instead of lineinfile.

---

Replaced with replace in 2be44f99